README.md

```diff
@@ -53,19 +57,21 @@ Dragonfly makes it simple and cost-effective to set up, operate, and scale any
 
 ## Comparison
 
-|Test Environment ||
+For Dragonfly, no matter how many clients start the file downloading, the average downloading time is almost stable without increasement (12s in experiment).
```

Comment:
12s? What's that mean?  Clould you please describe it more accurately, like the size of file or something others.